src/api/instances.rs

```diff
@@ -22,6 +22,7 @@ pub fn fetch_instances() -> std::result::Result<Vec<Instance>, reqwest::Error> {
             .linked
             .iter()
             .filter(|instance| instance.software == Some("lemmy".to_owned()))
```

Comment:
I think it would be better if we combine the two filter methods into one, due to performance reasons.

Reply:
Fixed: ee859e01c5ecff2e8f0a60e19f80c56a2caf92cd